src/base/base.pri

```diff
@@ -33,6 +33,7 @@ HEADERS += \
     $$PWD/bittorrent/magneturi.h \
     $$PWD/bittorrent/torrentinfo.h \
     $$PWD/bittorrent/torrenthandle.h \
+	$$PWD/bittorrent/torrentcategory.h \
```

Comment:
@glassez:  Is this indentation supposed to be different?

Reply:
Fixed in both places. Thanks.